package/CHANGELOG

```diff
@@ -65,9 +65,11 @@ Fixes
     argument. The directives parsed into bonds, angles, impropers, and dihedrals now
     match TPRParser. (PR #2408)
   * Added parmed to setup.py
+  * PersistenceLength.plot() now grab new axes if current axes not provided (Issue #2590)
```

Comment:
Put it under Enhancements, I wouldn't call it a "fix" – it improves the user experience but the problem did not lead to wrong results or crashes.

Reply:
Also add your gitgub handle at the end of the author list for "0.21" (if you are not there already)